src/modeler/AutoRest.Swagger/Validation/SummaryDescriptionValidation.cs

```diff
@@ -37,7 +37,7 @@ public class SummaryDescriptionValidation: TypedRule<Operation>
 
         public override IEnumerable<ValidationMessage> GetValidationMessages(Operation operation, RuleContext context)
         {
-            if(operation.Description != null && operation.Summary != null && operation.Description.Trim().Equals(operation.Summary.Trim(), StringComparison.CurrentCultureIgnoreCase))
+            if(operation.Description != null && operation.Summary != null && operation.Description.Trim().Equals(operation.Summary.Trim(), StringComparison.OrdinalIgnoreCase))
```

Comment:
null coalesce are used to return the values. Are you suggesting them to use here? I am not sure how much it will improve readability. Not changing now